app/javascript/groceries/components/check_in_items_list.vue

```diff
@@ -14,9 +14,10 @@ section(v-if="items.length > 0")
         :id='`trip-checkin-item-${item.id}`'
       )
       label.ml-2(:for='`trip-checkin-item-${item.id}`')
-        span.mr-1(:class='{ "text-gray-500": item.skipped }')
```

Comment:
This `mr-1` was causing the check-in content to be slightly horizontally scrollable when the right-margin was dangling at the end of the line and pushing the width of that `li` to be a bit wider than its container.